projects/angular-split/src/lib/directive/split-area.directive.ts

```diff
@@ -118,7 +118,7 @@ export class SplitAreaDirective implements OnInit, OnDestroy {
     })
 
     const iframeFixDiv = this.renderer.createElement('div')
-    this.renderer.addClass(iframeFixDiv, 'iframe-fix')
+    this.renderer.addClass(iframeFixDiv, 'as-iframe-fix')
```

Comment:
This a breaking change but seems more correct... @beeman worth it?

Reply:
It is a bit of a gray area... We never explicitly said this is public api. I think if we are going the deprecation route - just stay with it as it is now because it is not explicitly public api so no real need for the `as` prefix

Reply:
I would say, do the change. We can put a small piece of description in the release notes that we renamed a private part of the API. I would not count that as a breaking change. more a "breakingish?" one? (aka a `note` in the release notes)